### PR TITLE
Partial response

### PR DESCRIPTION
**Pull request by mkrump**

### CODE REVIEW

File: SwiftServer/Sources/FileSystem/FileSystem.swift [hunk 1]
```diff
@@ -19,7 +21,30 @@ public class ServerFileManager: FileSystem {
...
+
+    public func partialContents(atPath path: String, range: Range<Int>) -> Data? {
+        guard let size = fileSize(atPath: path) else {
+            return nil
+        }
+        if range.upperBound > size {
+            return nil
+        }
+        if let fileContents = FileManager.default.contents(atPath: path) {
+            return fileContents.subdata(in: range)
```
Comment (JamesChristie) [inline SwiftServer/Sources/FileSystem/FileSystem.swift:46]: I'm envious that it was this easy in your assigned language.

File: acceptance_tests.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ if [ ! -d $DIR/cob_spec ]; then
     mvn package
 fi
 cd $DIR/SwiftServer
-swift package clean
+#swift package clean
```
Comment (JamesChristie) [inline acceptance_tests.sh]: Why this change?

Comment (mkrump) [reply acceptance_tests.sh]: Just a mistake. Was taking too long to build cleanly while developing. Uncommented in e8d92e4
